js/parsers/paragraphparser.js

```diff
@@ -7,37 +7,66 @@ const pptelement_1 = require("airppt-models/pptelement");
  * Parse the paragraph elements
  */
 class ParagraphParser {
-    static extractParagraphElements(textElement) {
-        if (!textElement || !textElement["a:r"]) {
+    static extractParagraphElements(paragraphs) {
+        if (!paragraphs || paragraphs.length === 0) {
```

Comment:
paragraphs is set to `any` so best to check if this is array before checking `.length`

Reply:
paragarphs is set to `any[]` in ts. this is js file so maybe that's why you got confused

Reply:
ah yes, fine